quantum/keymap_extras/keymap_spanish_dvorak.h

```diff
@@ -119,7 +119,7 @@
 #define DV_CIRC S(DV_GRV)  // ^ (dead)
 #define DV_ASTR S(DV_PLUS) // *
 // Row 3
-#define DV_DIAE S(DV_GRV)  // ¨ (dead)
+#define DV_DIAE S(ES_ACUT)  // ¨ (dead)
```

Comment:
```suggestion
#define DV_DIAE S(DV_ACUT) // ¨ (dead)
```
And the `'` on line 29 should actually be `´`.

Reply:
my bad, sorry!